docs/app/pages/charts/charts-sections/timeline-chart/timeline-chart/timeline-chart.component.html

```diff
@@ -0,0 +1,108 @@
+<div class="main-chart">
+    <canvas baseChart
+            height="250"
+            [data]="lineChartData"
+            [options]="lineChartOptions"
+            [colors]="lineChartColors"
+            [legend]="false"
+            chartType="line">
+    </canvas>
+</div>
+
+<div class="timeline-chart">
+    <canvas baseChart
+            height="75"
+            [data]="timelineChartData"
+            [options]="timelineChartOptions"
+            [colors]="lineChartColors"
+            [legend]="false"
+            chartType="line">
+    </canvas>
+</div>
+
+<hr>
+
+<p>
+    Timeline Charts can be added using the <code>ng2-charts</code> library. The <code>chart.js</code> library needs to be imported and the <code>ChartsModule</code>
+    needs to be added to the appropriate <code>NgModule</code>.
+</p>
+<p>
+    To add Timeline functionality the <code>TimelineChartPlugin</code> should be imported from
+    the <code>@ux-aspects/ux-aspects</code> library and the <code>TimelineChartPlugin.register()</code> function should be called.
```

Comment:
Can you add some guidance on when/where the register function should be called? Is it something that can be done module-wide or in components only?